### DISCUSSION TITLE
Aid ! I can't do NEXT in a conversation

### DISCUSSION THREAD

**Discussion by juankstylez** [nutgram/nutgram#331] [General]
**Using** 

public function start(Nutgram $bot)
    {
        $bot->sendMessage('This is the first step!');
        $this->next('secondStep');
    }

    public function secondStep(Nutgram $bot)
    {
        $bot->sendMessage('Bye!');
        $this->end();
    }

    

By writing $this->next('xxx'); It gives me the following error :(

https://prnt.sc/j26Yp4xHAAzn

**I am calling it from another controller with an online button that appears in the chat**

https://prnt.sc/A1MWGqCz6Z2x

**Comment by Lukasss93** [comment]
Why are you using handlers inside handlers? It doesn't make sense, check the documentation carefully.

For your purpose, I recommend to use the InlineMenu:
https://nutgram.dev/docs/usage/inline_menu
